parachains/runtimes/collectives/collectives-polkadot/src/fellowship/migration.rs

```diff
@@ -0,0 +1,264 @@
+				);
+				return weight
+			}
+			let member_count = MemberCount::<T, I>::get(0);
+			weight.saturating_accrue(T::DbWeight::get().reads(1));
+			if member_count != 0 {
+				log::warn!(
+					target: TARGET,
+					"the collective already initialized, skipping import_fellowship migration."
+				);
+				return weight
+			}
+
+			let mut max_rank = 0;
+			for (rank, account_id32) in FellowshipAddresses::get() {
+				let who: T::AccountId = account_id32.into();
+				Members::<T, I>::insert(&who, MemberRecord::new(rank));
+				weight.saturating_accrue(T::DbWeight::get().writes(1));
+				for inner_rank in 0..rank + 1 {
+					let index = MemberCount::<T, I>::get(rank);
+					MemberCount::<T, I>::insert(inner_rank, index + 1);
+					IdToIndex::<T, I>::insert(inner_rank, &who, index);
+					IndexToId::<T, I>::insert(inner_rank, index, &who);
+					max_rank = max_rank.max(inner_rank);
+					// 2 writes to IdToIndex and IndexToId.
+					weight.saturating_accrue(T::DbWeight::get().writes(2));
+				}
+			}
+			// writes to MemberCount.
+			weight.saturating_accrue(T::DbWeight::get().writes((max_rank as u64) + 1));
+
+			weight
+		}
+
+		#[cfg(feature = "try-runtime")]
+		fn post_upgrade(_state: Vec<u8>) -> Result<(), &'static str> {
+			assert_eq!(MemberCount::<T, I>::get(0), 46, "invalid members count at rank 0.");
+			assert_eq!(MemberCount::<T, I>::get(1), 46, "invalid members count at rank 1.");
+			assert_eq!(MemberCount::<T, I>::get(2), 24, "invalid members count at rank 2.");
+			assert_eq!(MemberCount::<T, I>::get(3), 17, "invalid members count at rank 3.");
+			assert_eq!(MemberCount::<T, I>::get(4), 10, "invalid members count at rank 4.");
+			assert_eq!(MemberCount::<T, I>::get(5), 7, "invalid members count at rank 5.");
+			assert_eq!(MemberCount::<T, I>::get(6), 3, "invalid members count at rank 6.");
+			assert_eq!(MemberCount::<T, I>::get(7), 0, "invalid members count at rank 7.");
+			Ok(())
+		}
+	}
+}
+
+#[cfg(test)]
+pub mod tests {
+	use super::import_kusama_fellowship::FellowshipAddresses;
+	use crate::{FellowshipCollectiveInstance as Fellowship, Runtime, System};
+	use frame_support::traits::OnRuntimeUpgrade;
+	use pallet_ranked_collective::Rank;
+	use parachains_common::AccountId;
+	use sp_core::crypto::Ss58Codec;
+	use sp_runtime::AccountId32;
+
+	#[test]
+	fn check_fellowship_addresses() {
+		let fellowship_addresses = FellowshipAddresses::get();
+		let kusama_fellowship_ss58: [(Rank, _); 46] = [
+			(6, "16SDAKg9N6kKAbhgDyxBXdHEwpwHUHs2CNEiLNGeZV55qHna"), // proof https://kusama.subscan.io/extrinsic/16832707-4
+			(6, "J8ww78Qx3LVLW54bva3t4SzXcWKMdUWHEZR3V2VNKbmQgE8"),
+			(6, "FFFF3gBSSDFSvK2HBq4qgLH75DHqXWPHeCnR1BSksAMacBs"),
+			(5, "G7YVCdxZb8JLpAm9WMnJdNuojNT84AzU62zmvx5P1FMNtg2"),
+			(5, "15G1iXDLgFyfnJ51FKq1ts44TduMyUtekvzQi9my4hgYt2hs"), // proof https://kusama.subscan.io/extrinsic/16917610-2
+			(5, "Dcm1BqR4N7nHuV43TXdET7pNibt1Nzm42FggPHpxKRven53"),
+			(5, "J9nD3s7zssCX7bion1xctAF6xcVexcpy2uwy4jTm9JL8yuK"),
```

Comment:
```suggestion
			(5, "1363HWTPzDrzAQ6ChFiMU6mP4b6jmQid2ae55JQcKtZnpLGv"),
```

Polkadot address (known/verified on-chain). 

Here is the Kusama remark to match with https://kusama.subscan.io/extrinsic/16961180-2